holmes/plugins/prompts/generic_ask.jinja2

```diff
@@ -5,13 +5,43 @@ Do not say 'based on the tool output' or explicitly refer to tools at all.
 If you output an answer and then realize you need to call more tools or there are possible next steps, you may do so by calling tools at that point in time.
 If you have a good and concrete suggestion for how the user can fix something, tell them even if not asked explicitly
 
+CRITICAL: After calling tools, you MUST provide a clear, natural language answer to the user's question. Do not just output tool calls - always follow up with the actual answer or information the user requested.
```

Comment:
Can we merge kaito related prompt into one jinja file so that we can optionally turn these prompts on?

Reply:
Or we use a variable to toggle these prompt? Please forget my comments if these prompts are not specific to kaito